commons/push/integration/garmin/garmin_push_activity_details.avsc

```diff
@@ -0,0 +1,37 @@
+{
+  "namespace": "org.radarcns.push.integration.garmin",
```

Comment:
That this is an integration comes automatically. Would rename to `org.radarcns.push.garmin`